doc/man3/SSL_CTX_set_client_hello_cb.pod

```diff
@@ -0,0 +1,133 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_client_hello_cb, SSL_client_hello_cb_fn, SSL_client_hello_isv2, SSL_client_hello_get0_legacy_version, SSL_client_hello_get0_random, SSL_client_hello_get0_session_id, SSL_client_hello_get0_ciphers, SSL_client_hello_get0_compression_methods, SSL_client_hello_get1_extensions_present, SSL_client_hello_get0_ext - callback functions for early server-side ClientHello processing
+
+=head1 SYNOPSIS
+
+ #define SSL_CLIENT_HELLO_SUCCESS 1
+ #define SSL_CLIENT_HELLO_ERROR   0
+ #define SSL_CLIENT_HELLO_RETRY   (-1)
```

Comment:
These three need to go in the NAME section as well, per find-doc-nits.

Reply:
It then complains that they're not public, which seems to be about the .num files. Other constants like `SSL_READ_EARLY_DATA_ERROR` are not listed there or at the top of .pod files, so it looks like it'll be more consistent to remove them from this section? I've done that and doc-nits is happy now.

Reply:
util/private.num can contain entries for defines such as this that are public^Wdocumentable, if you can get your head around the naming of it.

Reply:
Yeah, it's a bad name.  My fault.  But more importantly, our style is not to list specific constant values in the documentation, but instead to refer to just the names in the prose.